sympy/combinatorics/fp_groups.py

```diff
@@ -69,13 +69,59 @@ def __new__(cls, fr_grp, relators):
         obj._is_standardized = False
 
         obj._order = None
+        obj._parent = None
         obj._center = None
         return obj
 
     @property
     def free_group(self):
         return self._free_group
 
+    @property
+    def parent(self):
+        return self._parent
+
+    @property
+    def relators(self):
+        return tuple(self._relators)
+
+    def _generators(self):
+        return self.free_group.generators
+
+    def subgroup(self, gens):
+        '''
+        Return the subgroup generated by `gens` using the
+        Reidemeister-Schreier algorithm
+
+        '''
+        if not all([isinstance(g, FreeGroupElement) for g in gens]):
+            raise ValueError("Generators must be `FreeGroupElement`s")
+        letters = [g.array_form[0][0] for g in self.generators]
```

Comment:
`self` is an `FpGroup` and it doesn't have `.symbols`. Though `self.free_group.symbols` is probably the same.